lib/appengine/util/gcloud.rb

```diff
@@ -162,7 +166,8 @@ def verify!
         #     depending on the value of the `capture` parameter.
         #
         def execute args, echo: false, capture: false, assert: true
-          joined_args = ::Shellwords.join args
+          Gem.win_platform? ? joined_args = args.join(" ") : joined_args = ::Shellwords.join(args)
```

Comment:
Actually, one other style nit: could you treat the ternary operator as an expression (as you do now in line 82 above) rather than a clumsy alternative to an "if" statement?

```ruby
joined_args = Gem.win_platform? ? args.join(" ") : ::Shellwords.join(args)
```

Reply:
You are correct actually, i'll use that style more in the future :) thanks.